frontend/public/models/index.ts

```diff
@@ -889,3 +889,54 @@ export const InfrastructureModel: K8sKind = {
   crd: true,
 };
 
+export const RevisionModel: K8sKind = {
```

Comment:
If we can plug into the console for knative, then these models should be defined in dev-console package.